torchao/quantization/marlin_qqq/README.md

```diff
@@ -0,0 +1,6 @@
+# Marlin QQQ
+
+Marlin QQQ implementation adapted from the two below sources:
```

Comment:
Could you summarize the key changes made from the Marlin kernel here?

Reply:
ok, I will do it these days.